docs/Netduino/Input_Output/Network/Configuration/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: Netduino
+title: Network Configuration
+subtitle: Configuring the Network interface
+---
+
+# Configuring Network Access
+
+The first step in accessing the network from a Netduino is to configure the network settings. Settings are stored directly on the device and are configured using the Netduino.Deploy tool.
+
+## Use Netduino.Deploy to configure network settings
```

Comment:
Should probably be Pascal Cased Headings. Also, i would drop "Use Netduino.Deploy" and just name it:

## Configure Network Settings

And then explain that it's done via Netduino.Deploy